README.md

```diff
@@ -129,21 +129,140 @@ Finally, run the replicator:
 poetry run cdf_fabric_replicator config.yaml
 ```
 
-# Docker
+## Visual Studio Code
+
+Alternatively, if you are using Visual Studio Code, just open the folder of the root directory of your project.
+
+You must still install Poetry as mentioned above in addition to the [Python Extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) for VSCode.
+
+The included ".vscode/launch.json" file will add "poetry install" and "poetry run" to your "Run and Debug" tab. They can be used in place of the manual command line statements above, and will attach the VSCode debugger.
+
+For more information see [Debugging in Visual Studio Code](https://code.visualstudio.com/Docs/editor/debugging).
+
+# Building and Deploying with Docker on AKS
+
+## Arm Architecture
 
 If you are running the Docker Container on ARM Architecture, you will need to use platform emulation for the Docker run command. You can do this by running the following command:
 
 ```
 docker run -i -t <image-name> --platform linux/arm64 
 ```
-=======
-## Visual Studio Code
 
-Alternatively, if you are using Visual Studio Code, just open the folder of the root directory of your project.
+## Creating an AKS(Azure Kubernetes Service) Cluster with Managed Identity and Attached ACR(Azure Container Registry)
 
-You must still install Poetry as mentioned above in addition to the [Python Extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) for VSCode.
+Azure Kubernetes Service (AKS) can use Azure Managed Identities to interact with other Azure services. This eliminates the need to manage service principals and rotate credentials. The Fabric Replicator requires managed identity on AKS to be enabled to run.
```

Comment:
Perhaps add this in a Requirements section with anything else needed to run the replicator (i.e. CDF, Fabric)

Reply:
We could but that would be something else we need to have a config for and manage that seemed out of scope for his story and PR

Reply:
I don't mind as long as its as simple as putting it all in a script with the variables such as the ACR Name the AKS Cluster Name and the App Name

Reply:
Added the Pre-Requisite Section